cards4-template.js

```diff
@@ -3,10 +3,39 @@ var makeCard = // receive factory with external name `makeCard`
 
```

Comment:
Overall, this is looking pretty good. Check below comments for some syntax issues and one function correction. If you have time, try to return to the makeCard.fullset and add functionality so it will create 52 cards; right now it's just creating an empty array.